src/test/ui/const-generics/issue-61336-2.rs

```diff
@@ -3,11 +3,12 @@
 
 fn f<T: Copy, const N: usize>(x: T) -> [T; N] {
     [x; {N}]
+    //~^ ERROR array lengths can't depend on generic parameters
```

Comment:
Huh I don't remember this error.

Reply:
Oh it's for `[x; n]` expressions, not types. Makes sense (although it should use a `ty::Const` instead of hardcoding an integer, cc @varkor @oli-obk ).